rpcs3/Emu/System.h

```diff
@@ -229,9 +233,10 @@ class Emulator final
 	static void CleanUp();
 
 	bool IsRunning() const { return m_state == system_state::running; }
-	bool IsPaused()  const { return m_state >= system_state::paused; } // ready is also considered paused by this function
+	bool IsPaused()  const { return m_state >= system_state::paused; } // ready/starting are also considered paused by this function
 	bool IsStopped() const { return m_state == system_state::stopped; }
 	bool IsReady()   const { return m_state == system_state::ready; }
+	bool IsStarting() const { return m_state == system_state::starting; }
```

Comment:
This whole logic is flawed, as almost every single place that checks for IsRunning would not also have to check if it's starting.

Maybe it would be better to add this state to IsRunning and make IsStarting a private function.